src/applications/disability-benefits/all-claims/Form526EZApp.jsx

```diff
@@ -255,4 +256,20 @@ const mapStateToProps = state => ({
   user: state.user,
 });
 
+if (isLoggedIn) {
+  datadogRum.init({
+    applicationId: 'c7eb541a-30d2-4a00-aba0-04965e8a2668',
+    clientToken: 'pub300747eeaef98ae4eb9c8d66f3c747c1',
+    site: 'ddog-gov.com',
+    service: 'benefits-526ez',
+    env: environment.vspEnvironment(),
+    sessionSampleRate: 10, // Set to 100 to test locally
+    sessionReplaySampleRate: 20, // Set to 100 to test locally
+    trackUserInteractions: true,
+    trackResources: true,
+    trackLongTasks: true,
+    defaultPrivacyLevel: 'mask-user-input',
+  });
+  datadogRum.startSessionReplayRecording();
+}
```

Comment:
I'd recommend initializing RUM replay recording behind a feature toggle. There's a lot of PII that'll show up in the recorded sessions until you find and mask/hide it. I spent quite a bit of time searching for and filtering out the PII in our relatively short form.

Reply:
Thanks @Mottie, this is great advice. I'm curious if you tried using `defaultPrivacyLevel: 'mask',` and if so if you still saw PII coming through?

Reply:
Mask only works for form elements, I think. We have a lot of pages that display content, so the class name is needed to mask/hide that content

Reply:
I went ahead and masked PII html data one by one, and not just input.  Just using 'mask' renders all text as XXX, example from datadog for reference.  
<img width="586" alt="image" src="https://github.com/department-of-veterans-affairs/vets-website/assets/6454840/9282022a-535c-4238-96f6-a591519a28dc">

Reply:
Do you have a specific need to see certain parts unmasked? I just worry about the possibility of fields getting added later on and not masked.

Reply:
From a quick glance at this PR, I think you're barely scratching the surface of all the PII - did you look at the review & submit page?

Reply:
I was thinking we'd lose context by masking all text.  @va-albers @Mottie what do you think?

@Mottie I'll give it another run through for PII that may show

Reply:
Sadly, you will lose context, but masking/hiding PII is important. Steve Albers had to check with platform security when I requested to keep the Country within the mailing address visible - it's not rendered when the Veteran selects United States; but it may help exposed issues with international addresses when they are used.